generics/services/sunbird-application.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * name : sunbird-application.js
+ * author : Rakesh Kumar
+ * Date : 02-Nov-2020
+ * Description : All sunbird service related information.
+ */
+
+//dependencies
+
+const request = require('request');
+
+
+
+/**
+  * Call to sunbird service. 
+  * @function
+  * @name callToSunbird
+  * @param requestBody - Logged in user Id.
+  * @param token - Logged in user token.
+  * @param url - url of the api call.
+  * @param requestType - http request method
+  * @returns {JSON} - sunbird service response
+*/
+
+function callToSunbird(requestType, url, token="", requestBody = "") {
```

Comment:
@rakeshSgr - Why is this service created when the sunbird service is already there?

Cc - @aman-tunerlabs - FYI

Reply:
sunbird.js is pointing to https://dev.bodh.shikshalokam.org" and sunbird-application.js pointing to our sunbird service application, so I kept different.